ERRATA.md

```diff
@@ -80,3 +80,20 @@ This document includes errata for the [Activity Streams](https://www.w3.org/TR/a
   - Unlike `latitude` and `longitude`, the domain of the `altitude` term is the `Object` type. The `altitude` term should be included in the list of properties of an `Object`. Because `altitude` is primarily documented as a property of a `Place`, publishers should not include `altitude` on objects that are not of type `Place`, and consumers should accept objects with this property that aren't of type `Place`.
 
   - The domain of the `attributedTo` property is both `Link` and `Object`. `attributedTo` should be included in the list of properties of a `Link`.
+
+  - Example 146 in Activity Vocabulary has an error in the format of the `startTime` property. The full example should be:
+  ```
+  {
+    "@context": "https://www.w3.org/ns/activitystreams",
+    "summary": "Sally became a friend of Matt",
+    "type": "Create",
+    "actor": "http://sally.example.org",
+    "object": {
+      "type": "Relationship",
+      "subject": "http://sally.example.org",
+      "relationship": "http://purl.org/vocab/relationship/friendOf",
+      "object": "http://matt.example.org",
+      "startTime": "2015-04-21T12:34:56Z"
```

Comment:
It seems odd to me that a friendship is assigned a start time *down to the second*. Perhaps it could be bumped to the nearest hour or even the nearest day, even if sometimes the nearest minute might be known...

Reply:
I think the resolution of startTime in the example is ok down to the second or even microsecond as long as it conforms to the range https://www.w3.org/TR/activitystreams-vocabulary/#dfn-starttime . The value generally reflects the moment that the  individual triggers a button while the server minding that the client's clock may not be in sync with server's clock.

That said, a Note or Considerations section could indicate:

* clock sync concern and any changes to the value after server's processing (possible error handling here if the value is beyond an acceptable or meaningful range) - this may need to be a separate issue if not already visited
* privacy concerns with respect to the resolution